helm/values.yaml

```
@@ -0,0 +1,88 @@
+# Default values for selfcare-external-api.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: selc
+
+replicaCount: 1
+
+image:
+  repository: selcdacr.azurecr.io/selfcareexternalapi
```

here should go the repo name without "-" (it's configured in gitops project)
```suggestion
  repository: selcdacr.azurecr.io/selfcareexternalapibackend
```